plugins/dell-dock/fu-dell-dock-usb-usb4.c

```diff
@@ -0,0 +1,603 @@
+/*
+ * Copyright (C) 2021 Intel Corporation.
+ * Copyright (C) 2021 Dell Inc.
+ * All rights reserved.
+ *
+ * This software and associated documentation (if any) is furnished
+ * under a license and may only be used or copied in accordance
+ * with the terms of the license.
+ *
+ * This file is provided under a dual MIT/LGPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
+ * Dell Chooses the MIT license part of Dual MIT/LGPLv2 license agreement.
+ *
+ * SPDX-License-Identifier: LGPL-2.1+ OR MIT
+ */
+
+#include "config.h"
+#include <string.h>
+#include <libfwupdplugin/fwupdplugin.h>
```

Comment:
I don't think you need the prefix of libfwupdplugin, just <fwupdplugin.h> is sufficient

Reply:
will move `fwupdplugin.h` to `fu-dell-dock-usb-usb4.h`